### PR TITLE
trguing for transmission: new mod creation

### PR DESCRIPTION
**Pull request by azlux**
Since the project transmission-web-control is archived, I've make a branch for the projet https://github.com/openscopeproject/TrguiNG as a mod.

Best regards,
Azlux

### CODE REVIEW

**Comment by azlux** [comment]
Test OK, should be good !

File: .github/workflows/BuildImage.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ on: [push, pull_request_target, workflow_dispatch]
 env:
   GITHUB_REPO: "linuxserver/docker-mods" #don't modify
   ENDPOINT: "linuxserver/mods" #don't modify
-  BASEIMAGE: "replace_baseimage" #replace
-  MODNAME: "replace_modname" #replace
+  BASEIMAGE: "transmission" #replace
+  MODNAME: "trguing" #replace
   MULTI_ARCH: "true" #set to false if not needed
```
Comment (aptalca) [inline .github/workflows/BuildImage.yml:10]: this should be false as the Dockerfile is not arch specific

**Review by aptalca — CHANGES_REQUESTED** [review]
service should depend on `init-mods` instead of `init-mods-package-install` as it doesn't install or need any packages

**Comment by azlux** [comment]
Sorry for all the small commits, I try to understand the usage of mods dependencies/scripts structure.

**Comment by aptalca** [comment]
no worries, it's fine

**Review by aptalca — APPROVED** [review]

**Pull request merged**